tests/integration/nm/ovs_test.py

```diff
@@ -98,7 +98,6 @@ def _create_bridge_iface(iface_bridge_settings):
 def _delete_iface(devname):
     nmdev = nm.device.get_device_by_name(devname)
     with mainloop():
-        nm.device.deactivate(nmdev)
```

Comment:
There is a typo in the commit message `ceanup`.

Reply:
Done